app/modules/advert/endpoints_advert.py

```diff
@@ -266,12 +266,9 @@ async def create_advert(
     except ValueError as error:
         raise HTTPException(status_code=400, detail=str(error))
     message = Message(
-        context=f"advert-new-{id}",
-        is_visible=True,
         title=f"📣 Annonce - {result.title}",
         content=result.content,
-        # The notification will expire in 3 days
-        expire_on=datetime.now(UTC) + timedelta(days=3),
+        action_module="advert",
```

Comment:
What would you think of using the module root?
```suggestion
        action_module=module.root,
```